janus/models/multiqc/models.py

```diff
@@ -419,7 +419,7 @@ class STARAlignment(BaseModel):
     unmapped_other: float
 
 
-class RNAfusionGeneralStats(BaseModel):
+class RNAFusionGeneralStatsForwardSummary(BaseModel):
```

Comment:
And the reverse?

Reply:
Changing for `RNAFusionGeneralStats`. Updating the one below with `RNAFusionReadStats` if @ChrOertlin approves

Reply:
Forward and reverse should have the same fields